api/doris/v1/types.go

```diff
@@ -88,6 +88,13 @@ type FeSpec struct {
 type BeSpec struct {
 	//the foundation spec for creating be software services.
 	BaseSpec `json:",inline"`
+
+	// EnableWorkloadGroup is a switch that determines whether the doris cluster enables the workload group.
+	// Default value is 'false'.
+	// Enabling it means that the container must be started in privileged mode.
+	// Please confirm whether the host machine and k8s cluster allow it.
+	// Doris workloadgroup reference document: https://doris.apache.org/docs/admin-manual/resource-admin/workload-group
+	EnableWorkloadGroup bool `json:"enableWorkloadGroup,omitempty"`
```

Comment:
the switch for doris means enable or not. for doris operator only represents kick-off privilege or not.
so, I think the field will be called Privileged, and reference the description to add comment.
	// Run container in privileged mode.
	// Processes in privileged containers are essentially equivalent to root on the host.
	// Defaults to false.
	// Note that this field cannot be set when spec.os.name is windows.
	// +optional